spec/traits.dd

```diff
@@ -403,6 +403,16 @@ $(H2 $(GNAME identifier))
 	$(P Takes one argument, a symbol. Returns the identifier
 	for that symbol as a string literal.
 	)
+---
+string fun() { return "abc"; }
+
+pragma(msg, fun); // "abc"
```

Comment:
I think this line should be removed, or at least explained, because if you're new to D it'll be very confusing as to why it prints "abc" and not "fun".